src/main/java/spoon/reflect/visitor/ImportScannerImpl.java

```diff
@@ -95,6 +93,7 @@
 		scan(invocation.getAnnotations());
 		scan(invocation.getTypeCasts());
 		scan(invocation.getTarget());
+		scan(invocation.getExecutable());
```

Comment:
This is needed to import actual type arguments, as in the added test -- without this line `java.io.File` won't be imported.